remo/profiles/migrations/0011_groups_new_onboarding_group.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
```

Comment:
The last migration in master branch is number `0009`. Probably you have one more migration from another branch.